Octokit/Models/Response/Issue.cs

```diff
@@ -64,7 +64,7 @@ public Issue(Uri url, Uri htmlUrl, Uri commentsUrl, Uri eventsUrl, int number, I
         public int Number { get; protected set; }
 
         /// <summary>
-        /// Whether the issue is open or closed.
+        /// Whether the issue is open, closed or all.
```

Comment:
all is not a valid option here so this comment can be reverted back to what it was